templates/ui-frameworks/lit/entry-type/ui/src/{{dna_role_name}}/{{coordinator_zome_manifest.name}}/{{kebab_case entry_type.name}}-detail.ts.hbs

```diff
@@ -17,7 +17,6 @@ import { decode } from '@msgpack/msgpack';
 import './edit-{{kebab_case entry_type.name}}';
 {{/if}}
 
-import { sharedStyles } from '../../shared-styles';
```

Comment:
Same here. It looks like this component uses the `alert` class for example which is defined in the shared styles so I would expect that this import should remain here.